kotlin/services/rds/Readme.md

```diff
@@ -1,23 +1,119 @@
+The credential provider used in all code examples is Shared credentials. For more information, see [Credential providers](https://docs.aws.amazon.com/sdk-for-kotlin/latest/developer-guide/credential-providers.html).
 
-You will find these examples: 
+To run these examples, set up your development environment to use Gradle to configure and build AWS SDK for Kotlin projects. For more information, 
+see [Get started with the SDK for Kotlin](https://docs.aws.amazon.com/sdk-for-kotlin/latest/developer-guide/get-started.html).
 
-- **CreateDBInstance** - Demonstrates how to create an Amazon RDS instance.
-- **CreateDBSnapshot** - Demonstrates how to create an Amazon RDS snapshot.
-- **DeleteDBInstance** - Demonstrates how to delete an Amazon RDS snapshot.
-- **DescribeAccountAttributes** - Demonstrates how to retrieve attributes that belong to an Amazon RDS account.
-- **DescribeDBInstances** - Demonstrates how to describe Amazon RDS instances.
-- **ModifyDBInstance** - Demonstrates how to modify an Amazon RDS instance.
+### Single actions
 
-To run these examples, you can setup your development environment to use Gradle to configure and build AWS SDK for Kotlin projects. For more information, 
-see [Get started with the AWS SDK for Kotlin](https://docs.aws.amazon.com/sdk-for-kotlin/latest/developer-guide/setup.html). 
+Code excerpts that show you how to call individual service functions using the **RdsClient** object: 
+
+- [Create an Amazon RDS instance and wait for it to be in an available state](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/kotlin/services/rds/src/main/kotlin/com/kotlin/rds/CreateDBInstance.kt) (createDBInstance command)
+- [Create an Amazon RDS snapshot](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/kotlin/services/rds/src/main/kotlin/com/kotlin/rds/CreateDBSnapshot.kt) (createDBSnapshot command)
+- [Delete an Amazon RDS instance](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/kotlin/services/rds/src/main/kotlin/com/kotlin/rds/DeleteDBInstance.kt) (deleteDBInstance command)
+- [Retrieve attributes that belong to an Amazon RDS account](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/kotlin/services/rds/src/main/kotlin/com/kotlin/rds/DescribeAccountAttributes.kt) (describeAccountAttributes command)
+- [Describe an Amazon RDS instance](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/kotlin/services/rds/src/main/kotlin/com/kotlin/rds/DescribeDBInstances.kt) (describeDBInstances command)
+- [Modify an Amazon RDS instance](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/kotlin/services/rds/src/main/kotlin/com/kotlin/rdsModifyDBInstance.kt) (modifyDBInstance command)
+
+### Scenario
+
```

Comment:
Code examples that show you how to accomplish a specific task by calling multiple functions within the same service.